### PR TITLE
Add Naval Flags Reference

### LINKED ISSUE #27: Add Signal Flag Reference
Add a reference for Naval Signal flags.

### PR DESCRIPTION
**Pull request by beckbria**
Fixes #27 
image

### CODE REVIEW

File: src/Views/Reference/NavalFlagTable.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      </div>
+    );
+  }
+
+  private renderRows(data: CharacterImage[]) {
+    return data.map((value: CharacterImage) => {
+      return (
+        <tr key={value.character}>
+          <td className="flagCharacter">{value.character}</td>
+          <td dangerouslySetInnerHTML={{__html: value.image.render(100)}} />
```
Comment (sjenks) [inline src/Views/Reference/NavalFlagTable.tsx]: Consider this instead of "dangerouslySetInnerHtml" https://www.npmjs.com/package/react-inlinesvg

Comment (kfarnung) [reply src/Views/Reference/NavalFlagTable.tsx]: I'd recommend using a [data URI](https://css-tricks.com/probably-dont-base64-svg/) instead.

Here's my proposed change: https://github.com/kfarnung/puzztool/commit/2c093365004fdb5e5a933e27113bfdc549c3964a

It requires https://github.com/kfarnung/puzzle-lib/pull/66 to land first though.

Comment (beckbria) [reply src/Views/Reference/NavalFlagTable.tsx]: Done.  We can potentially drop the width parameter from the render function entirely.

Comment (kfarnung) [reply src/Views/Reference/NavalFlagTable.tsx]: Opened a PR to drop the width parameter: https://github.com/kfarnung/puzzle-lib/pull/67

Suggested updates: https://github.com/kfarnung/puzztool/commit/b2ac463a9e1af704c3c1172b2d1d0810290ebd45

File: src/Views/Reference/NavalFlagTable.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            </tbody>
+          </Table>
+        </div>
+      </div>
+    );
+  }
+
+  private renderRows(data: CharacterImage[]) {
+    return data.map((value: CharacterImage) => {
+      const width = 100;
```
Comment (kfarnung) [inline src/Views/Reference/NavalFlagTable.tsx]: Add a className to the img tag and set this in CSS.  See https://github.com/kfarnung/puzztool/commit/b2ac463a9e1af704c3c1172b2d1d0810290ebd45

Comment (beckbria) [reply src/Views/Reference/NavalFlagTable.tsx]: Done.

**Comment by kfarnung** [comment]
Published a new version of puzzle-lib and updated my commit https://github.com/kfarnung/puzztool/commit/da00cec2922a8f680bf7ecc243d8771520b66af1

File: package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "ajv": "^6.5.4",
     "bootstrap": "^3.3.7",
-    "puzzle-lib": "^0.6.0",
+    "puzzle-lib": "^0.6.1",
```
Comment (kfarnung) [inline package.json]: 0.6.2 is ready

File: src/Views/Reference/NavalFlagTable.tsx (hunk 2 above)
Comment (kfarnung) [inline src/Views/Reference/NavalFlagTable.tsx]: You can get rid of the width if you upgrade to 0.6.2

**Pull request merged**